Tools/Pipeline/Common/PipelineTypes.cs

```diff
@@ -357,7 +357,7 @@ public static ImporterTypeDescription FindImporter(string name, string fileExten
 
             foreach (var i in Importers)
             {
-                if (i.FileExtensions.Contains(fileExtension))
+                if (i.FileExtensions.Contains(fileExtension.ToLowerInvariant()))
```

Comment:
How do we know `FileExtensions` is always lower case?  Do we force them to lowercase somewhere?